spec.html

```diff
@@ -404,7 +436,7 @@ <h1>function</h1>
       <h1>built-in function</h1>
       <p>built-in object that is a function</p>
       <emu-note>
-        <p>Examples of built-in functions include `parseInt` and `Math.exp`. An implementation may provide implementation-dependent built-in functions that are not described in this specification.</p>
+        <p>Examples of built-in functions include `parseInt` and `Math.exp`. A host may provide host-defined built-in functions that are not described in this specification.</p>
```

Comment:
As written, implementations are *not* hosts, because

> A host is an external source that further defines facilities listed in Annex D but **does not further define other implementation-defined or implementation-approximated facilities**. [emphasis added]

A strict reading of this sentence therefore does not permit the addition of new built-ins by implementations (though it also does not forbid them).

Perhaps it would be better to say

```suggestion
        <p>Examples of built-in functions include `parseInt` and `Math.exp`. A host or implementation may provide additional built-in functions that are not described in this specification.</p>
```

Reply:
Done.